CHANGELOG.md

```diff
@@ -7,6 +7,16 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 
 ## [Unreleased]
 
+### Changed
```

Comment:
can you add a section to the root Readme as we did before: https://github.com/giantswarm/prometheus-operator-app#from-13x-to-2x

So, another section
```
### from-2.x to 3.x
```

Reply:
done